bolt-simple-taproot.md

```diff
@@ -764,43 +764,50 @@ any keys aggregated via musig2 also are assumed to use the `KeySort` routine
 #### To Local Outputs
 
 For the simple taproot commitment type, we use the same flow of revocation or
-delay, but instead left the revocation case into the taproot key-spend case.
+delay, while re-using the NUMS point to ensure that the internal key is always
+revealed. This ensures that the anchor outputs can _always_ be spent given
+chain revelead information.
 
 The new output has the following form:
 
   * `OP_1 to_local_output_key`
   * where:
-    * `to_local_output_key = revocationpubkey + tagged_hash("TapTweak", revocationpubkey || to_delay_script_root)`
-    * `to_delay_script_root = tapscript_root([to_delay_script])`
+    * `to_local_output_key = revocationpubkey + tagged_hash("TapTweak", taproot_nums_point || to_delay_script_root)`
+    * `to_delay_script_root = tapscript_root([to_delay_script, revoke_script])`
     * `to_delay_script` is the delay script:
         ```
         <local_delayedpubkey> OP_CHECKSIG
         <to_self_delay> OP_CHECKSEQUENCEVERIFY OP_DROP
+    * `revoke_script` is the delay script:
+        ```
+        <revocation_pubkey> OP_CHECKSIG
+        ```
```

Comment:
I dont think this is correct. I think we want the script to include ` <local_delayedpubkey>` (just a push and drop of it) so that it is revealed no matter which branch is spent

Reply:
Yep, updating.